src/org/usfirst/frc/team166/robot/OI.java

```diff
@@ -4,27 +4,73 @@
 import edu.wpi.first.wpilibj.buttons.JoystickButton;
 
 import org.usfirst.frc.team166.robot.commands.DriveDistance;
+import org.usfirst.frc.team166.robot.commands.DriveWithJoysticks;
+import org.usfirst.frc.team166.robot.commands.DriveWithJoysticksReversed;
 
 /**
  * This class is the glue that binds the controls on the physical operator interface to the commands and command groups
  * that allow control of the robot.
  */
 public class OI {
 
+	private final boolean isReversed;
+
 	private final Joystick stickLeft;
 	private final Joystick stickRight;
 
-	private final Joystick testStick;
+	private final Joystick xbox;
 
 	public OI() {
+		isReversed = false;
 		stickLeft = new Joystick(RobotMap.stickLeft);
 		stickRight = new Joystick(RobotMap.stickRight);
 
-		testStick = new Joystick(0);
-
-		JoystickButton buttonA = new JoystickButton(testStick, 2);
+		xbox = new Joystick(0);
+
+		JoystickButton leftJoyTrigger = new JoystickButton(stickLeft, 1);
```

Comment:
I figured I should create them so that, when we did need to use them, we wouldn't have to keep going into OI and creating new buttons. If we don't use all of them, we can delete the ones we don't use.